src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -569,15 +585,15 @@ export function DashboardTopNav({
       createType: i18n.translate('dashboard.solutionToolbar.markdownQuickButtonLabel', {
         defaultMessage: 'Markdown',
       }),
-      onClick: createNewVisType('markdown'),
+      onClick: createNewVisType(markdownVisType),
       'data-test-subj': 'dashboardMarkdownQuickButton',
```

Comment:
Is there a `href` we can add here?

Reply:
It looks like `href` isn't support by `EuiButtonGroupOption`. The button group option only lets you define an `onClick` handler.